neo4j/pom.xml

```diff
@@ -34,6 +34,12 @@
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <!--
+                       see https://www.igorkromin.net/index.php/2017/08/17/how-to-enable-the-full-stack-trace-in-mavens-surefire-plugin-for-junit-testing/
+                       and
+                       http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#trimStackTrace
+                        -->
+                    <trimStackTrace>false</trimStackTrace>
```

Comment:
Please, add a small description and not only the link.

The website can change and we will never know what was pointing to.

Reply:
I also would love to have an idea of the change without having to surf the web

Reply:
Hi @DavideD .
I can read about the option at http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#trimStackTrace  and https://www.igorkromin.net/index.php/2017/08/17/how-to-enable-the-full-stack-trace-in-mavens-surefire-plugin-for-junit-testing/

Reply:
Maven plugin for junit cut stack traces. And it is not comfortable. The option configure the plugin for not cut stacktrces. It is very useful option